api/v1alpha1/volumereplicationgroup_types.go

```diff
@@ -75,6 +81,17 @@ type VolSyncReplicationDestinationSpec struct {
 	ProtectedPVC ProtectedPVC `json:"protectedPVC,omitempty"`
 }
 
+type PeerClass struct {
```

Comment:
We should add comments for new fields that are added, here are some suggestions:

// ReplicationID is the common value for the label "ramendr.openshift.io/replicationID" on the corresponding VolumeReplicationClass or VolumeGroupReplicationClass on each peer for the matched StorageClassName.

// StorageID is the collection of values for the label "ramendr.openshift.io/storageID" on the corresponding StorageClassName across the peers. It is singleton if the storage instance is shared across the peers, and distict if storage instances are different. 

// StorageClassName is the name of a StorageClass that is available across the peers

// PeerIDs are kubernetes cluster IDs, that share the common StorageClassName with the related parameters. Its value is typically the UID of the kube-system namespace of the clusters.

// DRPolicyStatus.Async contains the status of observed async replication details between the clusters in the policy

// DRPolicyStatus.Sync contains the status of observed sync replication details between the clusters in the policy

// DRPolicyStatus.[Async|Sync].PeerClasses is a list of common StorageClasses across the clusters in a policy that have related async or sync relationships. (one per pair of peers in the policy)

// VRG.Spec.[Async|Sync].PeerClasses is a list of common StorageClasses across the clusters in a policy that have related async or sync relationships. This is ONLY modified post creation, if the workload that is protected creates a PVC using a newer StorageClass that is determined to be common across the peers.

Reply:
done.